libs/design/badge/src/badge/badge.component.html

```diff
@@ -0,0 +1,3 @@
+<ng-content select="[daffBadgeIcon]"></ng-content>
+<ng-content select="[daffBadgeLabel]"></ng-content>
+<ng-content select="[daffBadgeDelete]"></ng-content>
```

Comment:
The delete icon should not be its own component. See `DaffNotificationComponent` for example of the `dismissible` property and how the template should be.

The disabled property can be moved up to the main badge component since the whole badge should be disableable, not just the delete button.